### PR TITLE
Separate multicluster profiles

### PR DESCRIPTION
**Pull request by frankbu**
Alternative to #3712

### CODE REVIEW

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md [hunk 1]
```diff
@@ -26,40 +26,64 @@ your specific needs. The following built-in configuration profiles are currently
...
+
+The `remote` profile is configured using the values file `values-istio-remove.yaml`. This profile installs only two
+Istio components:
+
+1. `istio-citadel` core component
+
+1. `istio-sidecar-injector` core component
+
+The `multicluster-gateways` profile is configured using the values file `values-istio-multicluster-gateways.yaml`.
+This profile installs the same commponents as the Istio `default` configuration profile plus two additional components:
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]:
```suggestion
This profile installs the same components as the Istio `default` configuration profile plus two additional components:
```

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md [hunk 2]
```diff
@@ -26,40 +26,64 @@ your specific needs. The following built-in configuration profiles are currently
...
+
+1. `istio-egressgateway` core component
+
+1. `corednsu` addon
```
Comment (linsun) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]: coredns

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md [hunk 3]
```diff
@@ -26,40 +26,64 @@ your specific needs. The following built-in configuration profiles are currently
...
+Istio provides two additional built-in configuration profiles that are used exclusively for configuring a
+[multicluster service mesh](/docs/concepts/multicluster-deployments/#multicluster-service-mesh):
+
+1. **remote**: used for configuring remote clusters of a
+    multicluster mesh with a [single control plane topology](/docs/concepts/multicluster-deployments/#single-control-plane-topology).
+
+1. **multicluster-gateways**: used for configuring all of the clusters of a
+    multicluster mesh with a [multiple control plane topology](/docs/concepts/multicluster-deployments/#multiple-control-plane-topology).
+
+The `remote` profile is configured using the values file `values-istio-remove.yaml`. This profile installs only two
```
Comment (linsun) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]: remove -> remote

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md [hunk 4]
```diff
@@ -26,40 +26,64 @@ your specific needs. The following built-in configuration profiles are currently
...
+Istio components:
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]:
```suggestion
Istio core components:
```

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md [hunk 5]
```diff
@@ -26,40 +26,64 @@ your specific needs. The following built-in configuration profiles are currently
...
+
+1. `istio-citadel` core component
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]:
```suggestion
1. `istio-citadel`
```

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md [hunk 6]
```diff
@@ -26,40 +26,64 @@ your specific needs. The following built-in configuration profiles are currently
...
+
+1. `istio-sidecar-injector` core component
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]:
```suggestion
1. `istio-sidecar-injector`
```

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md [hunk 7]
```diff
@@ -26,40 +26,64 @@ your specific needs. The following built-in configuration profiles are currently
...
+
+The `multicluster-gateways` profile is configured using the values file `values-istio-multicluster-gateways.yaml`.
+This profile installs the same commponents as the Istio `default` configuration profile plus two additional components:
+
+1. `istio-egressgateway` core component
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]:
```suggestion
1. The `istio-egressgateway` core component.
```

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md (hunk 2 above)
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]:
```suggestion
1. The `corednsu` addon.
```

**Pull request merged**